fixture-required.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
```

Comment:
There's a lot of fixtures now. Can you move them into a `test/fixtures` directory?